### PR TITLE
Add fields to capture email addresses of API owners that LGTM things.

### PR DESCRIPTION
**Pull request by jrobbins**
Add fields for i2e and i2s LGTMs to capture the email addresses of the API owners who approve these threads.

Eventually these might be automatically captured by having the API owners sign in and click a "LGTM" button in the tool themselves.  But, for now, we will ask feature developers to type in these email addresses from the thread.

Add progress detectors to set the checkmarks for these fields.

Also, for the existing spec process, there was no field to capture the URL of the intent to ship thread, so I added that.

### CODE REVIEW

**Comment by jrobbins** [comment]
@googlebot I consent.

**Comment by jrobbins-at-chromium** [comment]
@googlebot I consent

File: guideforms.py [hunk 1]
```diff
@@ -335,6 +335,20 @@
...
+        widget=forms.EmailInput(
+            attrs={'multiple': True, 'placeholder': 'email'}),
+        help_text=('Full email address of API owner who LGTM\'d the '
+                   'Intent to Experiment email thread.')),
+
+    'i2s_lgtms': forms.EmailField(
+        required=False, label='i2s LGTMs by',
+        widget=forms.EmailInput(
+            attrs={'multiple': True, 'placeholder': 'email, email, email'}),
+        help_text=('Full email addresses of API owner who LGTM\'d '
```
Comment (cwilso) [inline guideforms.py:349]: ownerS.

(Also, does this need to say comma-delimited?)

File: models.py [hunk 2]
```diff
@@ -1188,14 +1192,27 @@ class FeatureForm(forms.Form):
...
+      widget=forms.EmailInput(
+          attrs={'multiple': True, 'placeholder': 'email'}),
+      help_text=('Full email address of API owner who LGTM\'d the '
+                 'Intent to Experiment email thread.'))
+
+  i2s_lgtms = forms.EmailField(
+      required=False, label='i2s LGTMs by',
+      widget=forms.EmailInput(
+          attrs={'multiple': True, 'placeholder': 'email, email, email'}),
+      help_text=('Full email addresses of API owner who LGTM\'d '
```
Comment (cwilso) [inline models.py:1213]: owners.

**Comment by jrobbins** [comment]
I was thinking that users would need to type in email addresses for now.  I want to support an "LGTM" button in the tool, but I don't think that that can be done in a fast-follow.  It could end up in Q4.  I'll file an issue for it now.

File: guideforms.py [hunk 3]
```diff
@@ -335,6 +335,20 @@
         ('If your feature was available as an Origin Trial, link to a summary '
          'of usage and developer feedback. If not, leave this empty.')),
 
+    'i2e_lgtms': forms.EmailField(
+        required=False, label='i2e LGTM by',
+        widget=forms.EmailInput(
+            attrs={'multiple': True, 'placeholder': 'email'}),
+        help_text=('Full email address of API owner who LGTM\'d the '
```
Comment (jpmedley) [inline guideforms.py:342]: Who's supposed to enter them? The feature owner or the API owner who gave the LGTM?

File: processes.py [hunk 4]
```diff
@@ -203,10 +204,10 @@ def process_to_dict(process):
       'Prepare to ship',
       'Lock in shipping milestone. Finalize docs and announcements. '
       'Further standardization.',
-      ['Intent to Ship email',
-       'Three LGTMs',
-       'Documentation signoff',
+      ['Documentation signoff',
        'Updated target milestone',
+       'Intent to Ship email',
+       'Three i2s LGTMs',
```
Comment (jpmedley) [inline processes.py:210]:
```suggestion
       'Three LGTMs for the intent to ship',
```

Comment (jpmedley) [reply processes.py:210]: Do we have room to spell this out? Abbreviations of any kind are barriers to new participants, even if they seem obvious to us.

Comment (jpmedley) [reply processes.py:210]: If we have the room, I can change the others, but won't object if you get there first.

File: guideforms.py (hunk 1 above)
Comment (jrobbins) [reply guideforms.py:349]: Done.

File: models.py (hunk 2 above)
Comment (jrobbins) [reply models.py:1213]: Done.

File: guideforms.py (hunk 3 above)
Comment (jrobbins) [reply guideforms.py:342]: The feature owners for now.  Eventually we will have API owners click an LGTM button, but that will come later.

**Comment by jrobbins** [comment]
OK, I think I have addressed all the feedback.  Please take another look.

File: guideforms.py (hunk 3 above)
Comment (aglaforge) [reply guideforms.py:342]: My strong suspicion is that feature owners will not come back to the tool to do this.  It's manual work (copying and pasting from the email thread) that won't help them move the feature along (i.e., there's no enticement).

If we pick this path, over a checkbox, an LGTM button (even in a light weight form) will likely have to come pretty quickly.

File: processes.py (hunk 4 above)
Comment (jrobbins) [reply processes.py]: Done.

**Pull request merged**